pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBase.java

```diff
@@ -929,6 +935,9 @@ protected void callMessageListener(Message<T> msg) {
         } catch (Throwable t) {
             log.error("[{}][{}] Message listener error in processing message: {}", topic, subscription,
                     msg.getMessageId(), t);
+        } finally {
+            executorQueueSize.decrementAndGet();
+            triggerListener();
```

Comment:
Does this one will consume too many CPU cycles if the condition `executorQueueSize.get() < conf.getReceiverQueueSize()` = false for a while?

Reply:
I think it's the same behavior with the original. They are both consume too many CPU cycles in the first call of `triggerListener`